packages/components/src/navigation/style.scss

```diff
@@ -0,0 +1,34 @@
+.components-navigation {
+	width: 272px;
+	padding: 35px 30px;
+	background: #191e23;
+	color: #ccd0d4;
+}
+
+.components-navigation-back {
+	margin-bottom: 64px;
+	background-color: #23282d;
+	padding: 3px 6px;
+	color: #ccd0d4;
+}
+
+.components-navigation-title {
+	// For now
+	margin-bottom: 40px;
+}
+
+.components-navigation-items {
+	display: flex;
+	flex-direction: column;
+}
+
+.components-navigation-item {
+	display: flex;
+	justify-content: space-between;
+	margin-left: -12px;
+	color: #8f98a1;
+
+	&.is-active span {
```

Comment:
Should we also add a hover state for the font color change or do you think this is something we should override in the nav repo?

Reply:
I think all buttons will need their own color overrides to the default Button component in this repo. I think this can be a follow up.